.github/workflows/benchmarks.yml

```diff
@@ -143,18 +143,21 @@ jobs:
         if: always()
         run: |
           # Copy the full `asv continuous` log
-          cp asv_continuous.log .asv/results/
+          cp asv_continuous.log .asv/results/asv_continuous_${{ matrix.benchmark-name }}.log
+          # ensure that even if this isn't a PR, the benchmark_report workflow can run without error
+          touch .asv/results/message_${{ matrix.benchmark-name }}.txt
 
           # Add the message that might be posted as a comment on the PR
           # We delegate the actual comment to `benchmarks_report.yml` due to
           # potential token permissions issues
           if [[ $GITHUB_EVENT_NAME == pull_request ]]; then
 
           echo "${{ github.event.pull_request.number }}" > .asv/results/pr_number
-          cat <<EOF > .asv/results/message.txt
-          The benchmark run requested by $EVENT_NAME ($CONTENDER_REF vs $BASE_REF) has
-          finished with status `${{ steps.run_benchmark.outcome }}`. See the
+          cat <<EOF > .asv/results/message_${{ matrix.benchmark-name }}.txt
+          The ${{ matrix.benchmark-name }} benchmark run requested by $EVENT_NAME ($CONTENDER_REF vs $BASE_REF) has
+          finished with status '${{ steps.run_benchmark.outcome }}'. See the
           [CI logs and artifacts](||BENCHMARK_CI_LOGS_URL||) for further details.
+
           EOF
```

Comment:
Using `cat << EOF` creates newlines in the messages which looks weird on GitHub.

![Screen Shot 2022-11-08 at 7 55 11 AM](https://user-images.githubusercontent.com/2608297/200615758-7df94b50-ae07-403c-9091-a73fec19100e.png)

Can we just use `echo` with newlines in the code but not in the output?

```suggestion
          echo \
          "The ${{ matrix.benchmark-name }} benchmark run requested by $EVENT_NAME ($CONTENDER_REF vs $BASE_REF) has" \
          "finished with status '${{ steps.run_benchmark.outcome }}'. See the" \
          "[CI logs and artifacts](||BENCHMARK_CI_LOGS_URL||) for further details." \
          > .asv/results/message_${{ matrix.benchmark-name }}.txt
```

Reply:
Seems reasonable. I'll test and accept if it works as expected.